specs/packagemanager/PackageManagement.md

```diff
@@ -398,6 +400,24 @@ Is\*Provisioned\*() methods determine if the target is provisioned.
 
 These methods require administrative privileges.
 
+## 3.13. PackageValidator
+
+This API allows callers to verify that packages being processed by PackageDeploymentManager.AddPackageAsync
+and PackageDeploymentManager.StagePackageAsync match what are expected from their URI.
+
+When adding or staging a package from an external source such as HTTP URI or uncontrolled file location,
+an attacker can potentially intercept and tamper with the package data being read, causing a malicious
```

Comment:
CONSIDER: ...the package data being read (aka man-in-the-middle attacks), resulting in a malicious package...